pkg/providers/instancetype/instancetype.go

```diff
@@ -290,6 +311,30 @@ func processAvailableResources(az *ecsclient.DescribeAvailableResourceResponseBo
 	}
 }
 
+func getAllVSwitches(client *vpcclient.Client) ([]*vpcclient.DescribeVSwitchesResponseBodyVSwitchesVSwitch, error) {
+	var vSwitchInfo []*vpcclient.DescribeVSwitchesResponseBodyVSwitchesVSwitch
+	runtime := &util.RuntimeOptions{}
+	describeVSwitchesRequest := &vpcclient.DescribeVSwitchesRequest{
+		PageSize: tea.Int32(50),
+	}
+
+	for pageNumber := int32(1); pageNumber < 360; pageNumber++ {
```

Comment:
`360` feels too magic here. Is it coming from the limit from Aliyun or we are just assuming there won't be more than 18000 switches?

Reply:
The implementation is originally from `describeVSwitches` of vswitch.go, I think it just want to make sure all the items interested can be retrieved as intended, may be we can improve it a bit like fetching once to get the totalCount, and do for loop based on that?

Reply:
For vswitch, the number is not more than 10, let's just fetch them in one time